src/wallet.js

```diff
@@ -2894,6 +2894,11 @@ const wallet = {
 
     const accessData = this.getWalletAccessData();
     const encryptedXPriv = accessData.acctPathMainKey;
+
```

Comment:
I think we should just update the docstring to make it clear that the return can be `null`

Reply:
Done in 2717da25822ec6b1fb20eeaaf99381fcc0680638